.github/workflows/codeql.yml

```diff
@@ -0,0 +1,91 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Please take a look at other workflows to match the style a bit more. I made some comments here to give a direction to what I mean :)